lib/src/Uncertainty/Distribution/DistFunc.cxx

```diff
@@ -2139,7 +2139,7 @@ Sample DistFunc::rUniformSimplex(const Sample & vertices,
   Sample result(size, dimension);
   for (UnsignedInteger n = 0; n < size; ++n)
   {
-    result[n] = vertices[0];
+    for (UnsignedInteger j = 0; j < dimension; ++j) result(n, 0) = vertices(0, j);
```

Comment:
I dont understand this last change as the loop only alters one memory address, should it be result(n,j) ?

Reply:
If you're going to add another change, you can mention the new class in the changelog while you're at it

Reply:
indeed :-(